web/client/localConfig.json

```diff
@@ -308,7 +308,8 @@
           }, "AutoMapUpdate", "HelpLink", "DrawerMenu", "Version", "Notifications", "BackgroundSelector", "Swipe", {
             "name": "Annotations",
             "cfg": {
-              "measurementAnnotationEdit": false
+              "measurementAnnotationEdit": false,
+              "geodesic": true
```

Comment:
please remove these geodesic true from the config
the **true** should be a default value, not a config option